src/lang/fr.json

```diff
@@ -674,17 +674,17 @@
   "tier.order.product.title": "Commande",
   "tier.name.contribution": "contribution",
   "tier.name.payment": "paiement",
-  "updates.new.title": "New update",
+  "updates.new.title": "Une nouvelle mise à jour",
   "updates.create.login": "Vous devez vous identifier en tant qu'administrateur de ce collectif pour pouvoir créer un nouveau billet.",
   "expenses.byCategory": "Dépenses en {category}",
   "expenses.byRecipient": "Dépenses par {recipient}",
-  "error.code.invalid": "Invalid gift card code",
-  "redeem.subtitle.line1": "Open Collective helps communities - like open source projects, meetups, etc - raise money and operate transparently.",
+  "error.code.invalid": "Le code de carte-cadeau pas valable",
+  "redeem.subtitle.line1": "Open Collective aide des communautés - projets open-source, meetups etc - à collecter de l’argent et à opérer dans une façon transparente.",
```

Comment:
```suggestion
  "redeem.subtitle.line1": "Open Collective aide des communautés - projets open-source, meetups etc - à collecter de l’argent et à opérer de façon transparente.",
```

Reply:
Hi Betree! I tried to commit all your suggestions (thank you so much for checking btw!) but I'm know sure if I did it right. Did it work ??